Formula/dhall.rb

```diff
@@ -0,0 +1,26 @@
+require "language/haskell"
+
+class Dhall < Formula
+  include Language::Haskell::Cabal
+
+  desc "Compiler for the Dhall language"
+  homepage "https://github.com/dhall-lang/dhall-haskell"
+  url "https://hackage.haskell.org/package/dhall-1.18.0/dhall-1.18.0.tar.gz"
+  sha256 "b528ecff4f757fa085f61636ffb8a6eb90c761106d7d20bd735bee48a6abab94"
+  head "https://github.com/dhall-lang/dhall-haskell.git"
```

Comment:
Remove head. We only maintain it for widely-used formulas.

Reply:
Understood, removed.